packages/storybook-builder-vite/codegen-iframe-script.js

```diff
@@ -57,7 +57,7 @@ module.exports.generateIframeScriptCode =
         .map((entry) => `// preview entry\nimport '${entry}';`)
         .join('\n')} */
 
-    import { addDecorator, addParameters, addLoader, addArgTypesEnhancer, addArgsEnhancer } from '@storybook/client-api';
+    import { addDecorator, addParameters, addLoader, addArgTypesEnhancer, addArgsEnhancer, setGlobalRender } from '@storybook/client-api';
```

Comment:
It looks like `setGlobalRender` is not exported by `https://github.com/storybookjs/storybook/blob/c1befe4b2400920a6c52fdc947d8e598cc7e35e1/lib/client-api/src/index.ts#L1-L7`.

Reply:
OK that's a bug. Here's the actual line that does the work: 

https://github.com/storybookjs/storybook/blob/main/app/react/src/client/preview/index.tsx#L28